bib/internal/distrodef/distrodef.go

```diff
@@ -0,0 +1,60 @@
+package distrodef
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"strings"
+
+	"golang.org/x/exp/maps"
+	"gopkg.in/yaml.v3"
+)
+
+// ImageDef is a structure containing extra information needed to build an image that cannot be extracted
+// from the container image itself. Currently, this is only the list of packages needed for the installer
+// ISO.
+type ImageDef struct {
+	Packages []string `yaml:"packages"`
+}
+
+func loadFile(defDirs []string, distro string) ([]byte, error) {
+	for _, loc := range defDirs {
+		p := path.Join(loc, distro+".yaml")
+		if _, err := os.Stat(p); err == nil {
```

Comment:
This and the two lines below looks like a slightly incomplete cleanup. It works but is slightly confusing to read :)

Reply:
Oh my, a silly mistake, I pushed a fix.